grades/api.py

```diff
@@ -248,6 +248,46 @@ def generate_program_certificate(user, program):
     )
 
 
+def generate_program_letter(user, program):
+    """
+    Create a program letter if the user has a MM course certificate
+    for each course in the program and program is non-fa.
+
+    Args:
+        user (User): a Django user.
+        program (programs.models.Program): program where the user is enrolled.
+    """
+
+    if program.financial_aid_availability:
+        log.info(
+            'Congratulation letter can not be created for [%s], as given program [%s] if not non-fa',
```

Comment:
`'Congratulation letter is only available for non-financial aid programs'`. Also, I think this should be a `log.error`